packages/components/src/color-palette/index.js

```diff
@@ -2,7 +2,6 @@
  * External dependencies
  */
 import classnames from 'classnames';
-import { ChromePicker } from 'react-color';
```

Comment:
We need to remove `react-color` from `package.json` file as it is no longer used:

https://github.com/WordPress/gutenberg/blob/master/packages/components/package.json#L45

Reply:
fixed at 3c4686e